server/startup/migrations/v212.js

```diff
@@ -0,0 +1,21 @@
+import { MongoInternals } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 212,
+	up() {
+		const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
+		const apps = mongo.db.collection('rocketchat_apps');
+
+		Promise.await(apps.update({
```

Comment:
is it too late (or picky from me) to ask to change to `updateMany`? `update` is currently deprecated 😬

Reply:
I changed to `updateMany` =)